client/src/test/java/com/sinch/sdk/domains/verification/adapters/VerificationsStartServiceTest.java

```diff
@@ -35,7 +35,7 @@ public class VerificationsStartServiceTest extends BaseTest {
   @GivenJsonResource("/domains/verification/v1/start/StartVerificationFlashCallRequestDto.json")
   public StartVerificationRequestParameters startVerificationFlashCallRequestDto;
 
-  @GivenJsonResource("/domains/verification/v1/start/StartVerificationSeamlessRequestDto.json")
+  @GivenJsonResource("/domains/verification/v1/start/StartVerificationDataRequestDto.json")
   public StartVerificationRequestParameters startVerificationSeamlessRequestDto;
```

Comment:
Reviewing the commit related to seamless / data, you may also rename the variables here

Reply:
done